internal/util/k8s.go

```diff
@@ -63,8 +63,19 @@ func GetKubeNamespace() (string, error) {
 	return string(namespace), nil
 }
 
-func GetChaincodePodObject(imageData ImageJson, namespace, peerID string, chaincodeData ChaincodeJson) *apiv1.Pod {
-	chaincodeImage := imageData.Name + "@" + imageData.Digest
+func GetChaincodePodObject(devMode bool, imageData ImageJson, namespace, peerID string, chaincodeData ChaincodeJson) *apiv1.Pod {
+	var chaincodeImage string
+	var imagePullPolicy apiv1.PullPolicy
+	if devMode {
+		// Ignore the image digest in dev mode to allow chaincode which has not been pushed to
+		// a registry to start
+		// TODO log a suitable health warning!
+		chaincodeImage = imageData.Name + ":latest"
+		imagePullPolicy = apiv1.PullAlways
```

Comment:
Please remove the "Always" image pull policy.  In dev mode the user must be able to specify IfNotPresent.

Reply:
What's the use case for specifying `IfNotPresent` in a development mode?